docs/nnx/surgery.md

```diff
@@ -65,30 +65,30 @@ model.linear2 = model.linear1
 assert not hasattr(nnx.state(model), 'linear2')
 np.testing.assert_allclose(model(x), model.linear1(model.linear1(x)))
 
-# Variable sharing (weight tying)
+# Variable sharing (weight-tying)
 model = TwoLayerMLP(4, rngs=nnx.Rngs(0))
 model.linear1.kernel = model.linear2.kernel  # the bias parameter is kept separate
 assert hasattr(nnx.state(model), 'linear2')
 assert hasattr(nnx.state(model)['linear2'], 'bias')
 assert not hasattr(nnx.state(model)['linear2'], 'kernel')
 
-# Monkey patching
+# Monkey-patching
 model = TwoLayerMLP(4, rngs=nnx.Rngs(0))
 def awesome_layer(x): return x
 model.linear2 = awesome_layer
 np.testing.assert_allclose(model(x), model.linear1(x))
 
 ```
 
-## Create model and state without memory allocation
+## Creating and manipulating an abstract model or state without memory allocation
```

Comment:
This seems too verbose for a title. Can we drop "and manipulating"?

Reply:
SGTM!